android/gradle.properties

```diff
@@ -55,3 +55,7 @@ commitHash=unknown
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
 newArchEnabled=false
+
+# Use this property to enable or disable the Hermes JS engine.
+# If set to false, you will be using JSC instead.
+hermesEnabled=false
```

Comment:
Why are we disabling hermes?

Reply:
After bumping `react-native-reanimated` to v3 I was facing issues building the app when `hermes` was enabled.
now that things are a bit stable I will attempt to enable it in a follow up along with further research on moving to new architecture.